release/nightly_tests/dataset/dataset_ingest_400G_compute.yaml

```diff
@@ -0,0 +1,24 @@
+cloud_id: cld_17WvYIBBkdgLwEUNcLeRAE
```

Comment:
Why hardcoding the cloud id?

Reply:
ah, to resolve read from aws permission issues.